### ISSUE TITLE
Request for Training code release of 3D Attention branch

### ISSUE THREAD

**Issue by ajcanduyuu** [mazurowski-lab/SegmentAnyBone#18]
Hi, thank you for your wonderful work on this paper and for making the model publicly available![/+]
[+]The paper describes a novel 3D Attention Branch (Section 4.2.3 of the paper) that uses L3D as loss function.[/+]
[+]But I noticed in the current repository, the code for training the 3D Attention Branch does not seem to be included.[/+]
[+]Since the branch appears to be novel and important, I was wondering if you might be able to share the corresponding training code (or relevant scripts).[/+]
[+]In addition, I saw in your README that you plan to release code for training on other datasets by the end of this year. Would it be possible to upload just that portion a bit earlier, if feasible? Having access to this would greatly help with reproducing results and adapting SegmentAnyBone to new datasets in ongoing research.[/+]
[+]Thank you![/+]
